CONTRIBUTING.md

```diff
@@ -213,7 +213,7 @@ Facebook has a [bounty program](https://www.facebook.com/whitehat/) for the safe
 - Use [TypeScript](https://www.typescriptlang.org/).
 - Use semicolons;
 - Trailing commas,
-- Avd abbr wrds.
+- Avd abbr words.
```

Comment:
the point here is that the sentence is "avoid abbreviating words", so I think it should remain as is 🙂